src/js/panels.js

```diff
@@ -29,7 +29,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         },
         step: 0.1,
         selectors: {
-            instructions: ".gpiic-fd-instructions",
+            rangeInstructions: ".gpiic-fd-instructions",
```

Comment:
i think in the long term, components that reuse the range grade need to have its own protoTree block.